pulumi/bin/copy_artifacts_from_rc.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# This will grab a subset of keys from `origin/rc`'s Pulumi.testing.yaml.
+set -euo pipefail
+
+readonly THIS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+ensureRightDir() {
+    cd $THIS_DIR/../grapl
+}
+ensureRightDir
+
+readonly GRAPL_ROOT="$(git rev-parse --show-toplevel)"
+readonly RC_CONFIG_FILE="/tmp/rc_pulumi_testing.yaml"
+readonly CURRENT_STACK="$(pulumi stack --show-name)"
+
+confirmModify() {
+    read -r -p "This will modify your ${CURRENT_STACK} config. Continue (y/n)?" choice
+    case "${choice}" in
+        y | Y) echo "Okay!" ;;
+        *) exit 42 ;;
+    esac
+}
+
+add_artifacts() {
+    # Slightly tweaked version of what we have in
+    # .buildkite/shared/lib/rc.sh"
+    # removes the --cwd, --stack stuff
+    local -r stack="${1}"
+    local -r input_json="${2}"
+
+    source "${GRAPL_ROOT}/.buildkite/shared/lib/json_tools.sh"
+    flattened_input_json=$(flatten_json "${input_json}")
+
+    jq -r 'to_entries | .[] | [.key, .value] | @tsv' <<< "${flattened_input_json}" |
+        while IFS=$'\t' read -r key value; do
+            pulumi config set \
+                --path "artifacts.${key}" \
+                "${value}"
+        done
+}
+
+main() {
+    confirmModify
+
+    echo "--- Grab artifacts from origin/rc config file"
+    git fetch origin
+    git show origin/rc:pulumi/grapl/Pulumi.testing.yaml > ${RC_CONFIG_FILE}
```

Comment:
At some point the branch/tag and the stack will need to be parameters, but it's fine for now.